server/views/pages/overview/partials/workAndInterestsTabContents.njk

```diff
@@ -1,6 +1,6 @@
 {% if not workAndInterests.problemRetrievingData %}
 
-  {% if workAndInterest.data %}
+  {% if workAndInterests.data %}
```

Comment:
Also fixed a typo (without this fix it doesnt attempt to show any Work And Interests data from the CIAG response)